src/Simulation.h

```diff
@@ -4,12 +4,14 @@
 #include <memory>
 #include <any>
 
-#include "ensemble/CavityEnsemble.h"
 #include "io/TimerProfiler.h"
 #include "thermostats/VelocityScalingThermostat.h"
 #include "utils/FixedSizeQueue.h"
 #include "utils/FunctionWrapper.h"
 #include "utils/SysMon.h"
+#include "utils/Random.h"
+
+using Log::global_log;
```

Comment:
The `using Log::global_log;` issue is still open (see above in the PR description). It would be best to have it e.g. just once in the MarDyn.cpp file. But I think this is not working properly. Maybe also the Logger header class has to be included whenever `global_log` is used?

Reply:
Yeah I think the include is necessary, and I don't know if the `using` is necessary if the include is also used. But I think we should move this `using` back to the cpp file for the time being, and avoid all `using`s in header files. Maybe we can revisit this in a different PR if needed.

Reply:
Maybe @FG-TUM has a good idea on how to fix the `global_log` issue? It would be nice to solve it within this PR.

Reply:
Solution 2 sounds good to me. I just tried to implement it, but unfortunately I was not successful (it kind of compiled but said `undefined reference to 'global_log'`... Maybe you can give it a try @FG-TUM .

Reply:
I went for option 1 since it is easier and IMHO cleaner. The global changes I did in e6cd790 are (from the root folder):
- Remove all instances of the Log `using` declaration:
`sed --in-place 's/using Log::global_log;//g' /src/**/*.(cpp|h)`
- Replace all occurrences of `global_log` with `Log::global_log`:
`sed --in-place 's/\([\t ()]\)\(global_log\)/\1Log::\2/g' src/**/*.(cpp|h)`

Hope this caught everything :shrug: